src/sentry/static/sentry/app/views/issueList/overview.tsx

```diff
@@ -1066,7 +1066,6 @@ class IssueListOverview extends React.Component<Props, State> {
                     tags={tags}
                     query={query}
                     onQueryChange={this.onIssueListSidebarSearch}
-                    orgId={orgSlug}
```

Comment:
This was being masked by the `propType` type checking being lax. This prop does not exist and is not used on IssueListSidebar